### PR TITLE
FuelType and PropulsionType unbounded

### PR DESCRIPTION
**Pull request by ue71603**
fixes: https://github.com/NeTEx-CEN/NeTEx/issues/561

### CODE REVIEW

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 1]
```diff
@@ -194,20 +194,20 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Whether vehicles of the type are self-propelled.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
-			<xsd:element name="PropulsionType" type="PropulsionTypeEnumeration" minOccurs="0">
+			<xsd:element name="PropulsionType" type="PropulsionTypeEnumeration" minOccurs="0" maxOccurs="unbounded">
```
Comment (Aurige) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:197]: we could have used a list of enumeration (if not we may need a wrapper tag)

        _<xsd:simpleType name="PropulsionTypeListOfEnumerations">
		<xsd:annotation>
			<xsd:documentation>List of Allowed values for propulsion type.</xsd:documentation>
		</xsd:annotation>
		<xsd:list itemType="PropulsionTypeEnumeration"/>
	</xsd:simpleType>_

**Review by trurlurl — APPROVED** [review]

**Review by skinkie — APPROVED** [review]

**Review by Aurige — APPROVED** [review]

**Review by nick-knowles — COMMENTED** [review]
NeTEx UML Updated
image
Transmodel UML updated
image

**Comment by ue71603** [comment]
@nick-knowles  thanks for updating the diagrams

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd (hunk 1 above)
Comment (ue71603) [reply xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd]: done

**Pull request merged**